src/wazuh_db/wdb.c

```diff
@@ -1255,3 +1269,17 @@ void wdb_free_agent_info_data(agent_info_data *agent_data) {
         os_free(agent_data);
     }
 }
+
+sqlite3_stmt* wdb_start_cached_transaction(wdb_t* wdb, wdb_stmt statement_index){
```

Comment:
Maybe the **cached** word in the name suggests that this functions doesn't perform the `cache` operation

Reply:
Renamed to **wdb_init_stmt_in_cache**